lib/game.js

```diff
@@ -0,0 +1,75 @@
+const { GameEngine, VsStepper } = require('./engine');
+
+
+class Game {
+  constructor(server) {
+    this.server = server;
+    this.players = [];
+    this.engine = new GameEngine({
+      stepper: VsStepper,
+      flashTime: 40,
+      floatTime: 10,
+      swapTime: 3,
+      garbageFlashTime: 2,
+      blockTypes: [
+        'red',
+        'gold',
+        'lawngreen',
+        'darkcyan',
+        'blue',
+        'blueviolet',
+      ],
+    });
+  }
+
+  addPlayer(player) {
+    if (this.players.length > 1) {
+      return;
+    }
+    player.game = this;
+    this.players.push(player);
+    if (this.players.length === 2) {
```

Comment:
The `2` here should be a module level constant and should be added to options as `numPlayers` to be explicit about the defaults.